### DISCUSSION TITLE
How to reset main administrator user

### DISCUSSION THREAD

**Discussion by Mindaugas1983** [Budibase/budibase#7901] [Help]
I had created admin user in freshly installed budibase and cannot login into that account. How can i reset admin user?  I am getting error that user does not exists...

**Comment by melohagan** [comment]
Is this via **docker-compose** ? You'll need to nuke the docker volumes to clear the Budibase databases.

They'll most likely be called something like this:
![Screenshot 2022-09-27 at 09 54 59](https://user-images.githubusercontent.com/101575380/192481063-8ad70ef2-5092-49c9-8a2e-65023cc499e3.png)

You can find them by running: `docker volume ls`

Then to remove them, run `docker volume rm budibase_couchdb3_data` (and again for the other two).

Then run through the installation again and it should be a clean start!
